<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameWindow</class>
 <widget class="QMainWindow" name="GameWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>817</width>
    <height>628</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Minesweeper</string>
  </property>
  <widget class="QWidget" name="root">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#root_grid {
	border: 0px solid gray;
    border-right: 4px solid gray;
	border-bottom: 4px solid gray;
	border-top: 4px solid white;
	border-left: 4px solid white;
    background-color: rgb(205, 205, 205);
}

QWidget#board_widget {
	margin: 0px 11px 11px 11px;
	border: 0px solid gray;
    border-top: 4px solid gray;
	border-left: 4px solid gray;
	border-right: 4px solid white;
	border-bottom: 4px solid white;
    background-color: rgb(205, 205, 205);
    color: rgb(0, 0, 0);
}

QWidget#control_widget {
	margin: 11px 11px 11px 11px;
	border: 0px solid gray;
    border-top: 4px solid gray;
	border-left: 4px solid gray;
	border-right: 4px solid white;
	border-bottom: 4px solid white;
    background-color: rgb(205, 205, 205);
    color: rgb(0, 0, 0);
}

QPushButton#ctrl_button_restart {
    border: 0px solid gray;
    border-right: 4px solid gray;
	border-bottom: 4px solid gray;
	border-top: 4px solid white;
	border-left: 4px solid white;
    background-color: rgb(205, 205, 205);
	margin-top: 19px;
	margin-bottom: 19px;
    color: rgb(0, 0, 0);
}
        
QPushButton#ctrl_button_restart:pressed {
	border: 0px solid gray;
    border-right: 4px solid white;
	border-bottom: 4px solid white;
	border-top: 4px solid gray;
	border-left: 4px solid gray;
    background-color: rgb(205, 205, 205);
    color: rgb(0, 0, 0);
}

QWidget#window_bar {
    margin: 4px 6px 0px 6px;
    background-color: rgb(0, 0, 148);
}

QPushButton#window_close {
    border: 0px solid gray;
    border-right: 2px solid gray;
	border-bottom: 2px solid gray;
	border-top: 2px solid white;
	border-left: 2px solid white;
    background-color: rgb(205, 205, 205);
    color: rgb(0, 0, 0);
}
        
QPushButton#window_close:pressed {
	border: 0px solid gray;
    border-right: 2px solid white;
	border-bottom: 2px solid white;
	border-top: 2px solid gray;
	border-left: 2px solid gray;
    background-color: rgb(205, 205, 205);
    color: rgb(0, 0, 0);
}

QPushButton#window_min {
    border: 0px solid gray;
    border-right: 2px solid gray;
	border-bottom: 2px solid gray;
	border-top: 2px solid white;
	border-left: 2px solid white;
    background-color: rgb(205, 205, 205);
    color: rgb(0, 0, 0);
}
        
QPushButton#window_min:pressed {
	border: 0px solid gray;
    border-right: 2px solid white;
	border-bottom: 2px solid white;
	border-top: 2px solid gray;
	border-left: 2px solid gray;
    background-color: rgb(205, 205, 205);
    color: rgb(0, 0, 0);
}

.QToolButton {
    margin-top: 2px;
    border: 0px solid black;
    color: black;
}

QLabel#window_title {
    margin: 0px 0px 2px 2px;
    color: rgb(255, 255, 255);
}</string>
   </property>
   <layout class="QGridLayout" name="root_layout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QWidget" name="root_grid" native="true">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <layout class="QGridLayout" name="root_grid_layout" rowstretch="0,0,0,0">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QWidget" name="window_bar" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <property name="leftMargin">
           <number>6</number>
          </property>
          <property name="topMargin">
           <number>8</number>
          </property>
          <property name="rightMargin">
           <number>12</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item row="0" column="3">
           <widget class="QPushButton" name="window_close">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>23</width>
              <height>23</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>23</width>
              <height>23</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="iconSize">
             <size>
              <width>15</width>
              <height>15</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <spacer name="window_spacer">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Policy::MinimumExpanding</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="window_title">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>26</height>
             </size>
            </property>
            <property name="text">
             <string>Minesweeper</string>
            </property>
            <property name="textFormat">
             <enum>Qt::TextFormat::PlainText</enum>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QPushButton" name="window_min">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>23</width>
              <height>23</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>23</width>
              <height>23</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="iconSize">
             <size>
              <width>15</width>
              <height>15</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QWidget" name="board_widget" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font/>
         </property>
         <layout class="QGridLayout" name="board_widget_layout">
          <property name="leftMargin">
           <number>15</number>
          </property>
          <property name="topMargin">
           <number>8</number>
          </property>
          <property name="rightMargin">
           <number>15</number>
          </property>
          <property name="bottomMargin">
           <number>24</number>
          </property>
          <property name="spacing">
           <number>0</number>
          </property>
         </layout>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QWidget" name="control_widget" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>78</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>75</height>
          </size>
         </property>
         <layout class="QGridLayout" name="control_widget_layout" columnstretch="0,0,0,0">
          <property name="leftMargin">
           <number>16</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>16</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item row="0" column="3">
           <spacer name="ctrl_spacer_right">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Policy::Expanding</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="2">
           <widget class="QPushButton" name="ctrl_button_restart">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>40</width>
              <height>50</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>92</width>
              <height>1000</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="iconSize">
             <size>
              <width>27</width>
              <height>27</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <spacer name="ctrl_spacer_left">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QWidget" name="menu_bar" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="leftMargin">
           <number>6</number>
          </property>
          <property name="topMargin">
           <number>6</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QToolButton" name="menu_game">
            <property name="text">
             <string>Game</string>
            </property>
            <property name="popupMode">
             <enum>QToolButton::ToolButtonPopupMode::InstantPopup</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="menu_help">
            <property name="font">
             <font>
              <pointsize>15</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Help</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="menu_spacer_right">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
